features/util.py

```diff
@@ -151,6 +153,7 @@ def repo_state_hash(
 
 def build_debs(
     series: str,
+    architecture: str = get_dpkg_arch(),
```

Comment:
Currently, passing `architecture=None` like we do down in build.py will overwrite this default value and then we'll get an error trying to create the sbuild command. We can do something like the following to allow the build.py to work:
```suggestion
    architecture: Optional[str] = None,
```
and then
```python
if architecture is None:
    architecture = get_dpkg_arch()
```

Reply:
Overlooked that - fixed